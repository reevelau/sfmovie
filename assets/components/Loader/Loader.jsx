import React, { Component } from 'react';

export default class Loader extends Component {
  constructor(props) {
    super(props);
  }

  render() {
    return (
      <div class="loader-layer">
        <div class="loader"></div>
        <div class="loader-message">{this.props.loadingMessage}</div>
      </div>
    );
  }
}