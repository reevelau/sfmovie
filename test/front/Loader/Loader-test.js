import React from 'react';
import Loader from '../../../assets/components/Loader/Loader';
import renderer from 'react-test-renderer';


test('Normal', () => {

    const component = renderer.create(
        <Loader loadingMessage="We need few seconds to be well prepared." />
    );
    
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
});