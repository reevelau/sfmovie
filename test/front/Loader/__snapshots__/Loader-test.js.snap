// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Normal 1`] = `
<div
  className="loader-layer"
>
  <div
    className="loader"
  />
  <div
    className="loader-message"
  >
    We need few seconds to be well prepared.
  </div>
</div>
`;
